analytic_engine/src/instance/flush_compaction.rs

```diff
@@ -1062,6 +1079,40 @@ impl SpaceStore {
     }
 }
 
+/// Collect the column stats from a batch of sst meta data.
+fn collect_column_stats_from_meta_datas(metas: &[SstMetaData]) -> HashMap<String, ColumnStats> {
+    let mut low_cardinality_counts: HashMap<String, usize> = HashMap::new();
```

Comment:
Could we init its capacity? we have very wide table in prod.

Reply:
There is no entry for the columns whose cardinality is high, so I guess in the most of cases there is no need to do memory allocation.

Reply:
That's a good point.